crates/re_ui/Cargo.toml

```diff
@@ -29,7 +29,7 @@ default = []
 re_entity_db.workspace = true # syntax-highlighting for InstancePath. TODO(emilk): move InstancePath
 re_log_types.workspace = true # syntax-highlighting for EntityPath
 
-egui_commonmark.workspace = true
+egui_commonmark = { workspace = true, features = ["pulldown_cmark"] }
```

Comment:
the latest version requires us to pick one -- this is the default

Reply:
https://github.com/lampsitter/egui_commonmark/blob/master/Backends.md

> pulldown_cmark has fewer dependencies and should theoretically be faster

Sounds like `pulldown_cmark` is worth a try